woocommerce/woocommerce-android
# [Login] Allow installing Woo for non-woo sites

Closes: #7073 

### Description
This PR adds the ability to install Woo during the login with site address flow if the site doesn't have it, it uses an authenticated WebView for handling the installation.

### Testing instructions
1. You need a site that has Jetpack connected but doesn't have Woo.
2. Open the app, and login using the site address.
3. When you get to the "Woo not found" error screen, tap on "Install WooCommerce"
4. Confirm this opens a WebView where you can install WooCommerce.
5. Click on "Install and Activate"
6. Wait for the installation to finish.
7. Confirm the app closes the webview when the installation finishes.
8. Confirm the login continues successfully.

### Images/gif
https://user-images.githubusercontent.com/1657201/182592473-59da8553-31da-4a96-a200-9a0af55c591e.mov
